### PR TITLE
Adding sitting and love to emote app

### PR DESCRIPTION
**Pull request by alexiamandeville**
1. Adding sitting emotes to emote app (animations, HTML)
2. Adding love emote to emote app (animations, HTML)

These changes relate to this document: https://highfidelity.atlassian.net/wiki/spaces/DES/pages/283607105/Sitting+Improvements

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9186/app_13003_PR_36df9d7de86a067ef045c459403b7b5cb1041430.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9186/HighFidelity-Beta-PR13003-36df9d7de86a067ef045c459403b7b5cb1041430.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9186/HighFidelity-Beta-Interface-PR13003-36df9d7de86a067ef045c459403b7b5cb1041430.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9186/HighFidelity-Beta-PR13003-36df9d7de86a067ef045c459403b7b5cb1041430.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9186/HighFidelity-Beta-Interface-PR13003-36df9d7de86a067ef045c459403b7b5cb1041430.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10103/HighFidelity-Beta-PR13003-81d615c9db3c65ef312edd02bfeb6c625c384836.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10103/HighFidelity-Beta-PR13003-81d615c9db3c65ef312edd02bfeb6c625c384836.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10103/HighFidelity-Beta-Interface-PR13003-81d615c9db3c65ef312edd02bfeb6c625c384836.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10103/HighFidelity-Beta-PR13003-81d615c9db3c65ef312edd02bfeb6c625c384836.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10103/HighFidelity-Beta-Interface-PR13003-81d615c9db3c65ef312edd02bfeb6c625c384836.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10111/HighFidelity-Beta-PR13003-554056173025459011000521a09b4e90501c8803.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10111/HighFidelity-Beta-PR13003-554056173025459011000521a09b4e90501c8803.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10111/HighFidelity-Beta-Interface-PR13003-554056173025459011000521a09b4e90501c8803.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10111/HighFidelity-Beta-PR13003-554056173025459011000521a09b4e90501c8803.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10111/HighFidelity-Beta-Interface-PR13003-554056173025459011000521a09b4e90501c8803.exe

File: scripts/system/emote.js [hunk 1]
```diff
@@ -71,34 +72,77 @@ function onWebEventReceived(event) {
...
+
+        
     }
 }
 
+function restoreAnimation() {
+    MyAvatar.restoreAnimation();
+}
+
+Controller.keyPressEvent.connect(restoreAnimation);
```
Comment (ZappoMan) [inline scripts/system/emote.js]: I assume the purpose of this code is to allow any button mash to stop the animation, can you add a comment stating as such.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10126/HighFidelity-Beta-PR13003-cdf56d3c6d40e22e1c5f67ea58bcbcf50951e838.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10126/HighFidelity-Beta-PR13003-cdf56d3c6d40e22e1c5f67ea58bcbcf50951e838.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10126/HighFidelity-Beta-Interface-PR13003-cdf56d3c6d40e22e1c5f67ea58bcbcf50951e838.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10126/HighFidelity-Beta-PR13003-cdf56d3c6d40e22e1c5f67ea58bcbcf50951e838.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10126/HighFidelity-Beta-Interface-PR13003-cdf56d3c6d40e22e1c5f67ea58bcbcf50951e838.exe

**Comment by conklin94122** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10586/HighFidelity-Beta-PR13003-cdf56d3.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10586/HighFidelity-Beta-PR13003-cdf56d3.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10586/HighFidelity-Beta-Interface-PR13003-cdf56d3.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10586/HighFidelity-Beta-PR13003-cdf56d3.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10586/HighFidelity-Beta-Interface-PR13003-cdf56d3.exe

**Comment by qliejahner** [comment]
Currently testing

**Comment by qliejahner** [comment]
Pass @conklin94122 

This is likely outside the scope of this PR, but it's a bit awkward to remain at the same height in VR while the avatar sits down below you.

Moving is a little silly as well:

hifi-snap-by--on-2018-06-14_12-40-47

**Pull request merged**